installscripts/jazz-terraform-unix-noinstances/chefProvisioner.tf

```diff
@@ -79,12 +79,12 @@ resource "null_resource" "chef_provision_jenkins_server" {
   provisioner "remote-exec" {
     inline = [
       "sudo sh ~/cookbooks/installChef.sh",
-      "curl -L https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 -o /usr/local/bin/jq",
+      "sudo curl -L https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 -o /usr/local/bin/jq",
```

Comment:
Good catch, this `sudo` is required because we are `curl`-ing the file to a system bin folder.